Tools/TestWebKitAPI/Tests/WebKitCocoa/ElementTargetingTests.mm

```diff
@@ -351,6 +351,12 @@ @implementation _WKTargetedElementInfo (TestingAdditions)
         EXPECT_TRUE([adjustedSelectors containsObject:@".absolute.bottom-right"]);
         EXPECT_TRUE([adjustedSelectors containsObject:@".fixed.container"]);
         EXPECT_TRUE([adjustedSelectors containsObject:@".absolute.bottom-left"]);
+
+        [webView objectByEvaluatingJavaScript:@"[...document.querySelectorAll('.fixed,.absolute')].map(e => e.style.display = 'none')"];
+        [webView waitForNextPresentationUpdate];
+        EXPECT_GT([webView numberOfVisibilityAdjustmentRects], 0U);
+        [webView objectByEvaluatingJavaScript:@"[...document.querySelectorAll('.fixed,.absolute')].map(e => e.style.display = '')"];
+        [webView waitForNextPresentationUpdate];
```

Comment:
Did you mean to assert `EXPECT_EQ(..., 0U)` (though I suppose that's already covered elsewhere) or just reset the `display` property?

Reply:
Oh, this second script evaluation is just to reset the display property after checking for ≥1 rects.